baybe/utils/dataframe.py

```diff
@@ -418,3 +418,23 @@ def fuzzy_row_match(
             inds_matched.extend(inds_found)
 
     return pd.Index(inds_matched)
+
+
+def pretty_print_df(df: pd.DataFrame) -> str:
+    """Convert a dataframe into a pretty/readable str object.
```

Comment:
`str object` sounds weird to me, I would just be general here and say "format".

Reply:
Fixed. Thank you!